modules/fscloud/variables.tf

```diff
@@ -3,50 +3,50 @@
 ##############################################################################
 
 variable "resource_group_id" {
-  description = "ID of resource group to use when creating the event stream instance"
+  description = "The resource group ID to use when creating the Event Streams instance."
   type        = string
 }
 
 variable "name" {
   type        = string
-  description = "The name to give the IBM Event Notification instance created by this module."
+  description = "The name of the Event Notifications instance that is created by this module."
 }
 
 variable "tags" {
   type        = list(string)
-  description = "Optional list of tags to be added to the Event Notification instance"
+  description = "The optional list of tags to add to the Event Notifications instance."
   default     = []
 }
 
 variable "region" {
   type        = string
-  description = "IBM Cloud region where event notification will be created, supported regions are: us-south (Dallas), eu-gb (London), eu-de (Frankfurt), au-syd (Sydney), eu-es (Madrid)"
+  description = "The IBM Cloud region where the Event Notifications resource is created. The following regions are supported: `us-south` (Dallas), `eu-gb` (London), `eu-de` (Frankfurt), `au-syd` (Sydney), `eu-es` (Madrid)"
   default     = "us-south"
 }
 
 variable "skip_iam_authorization_policy" {
   type        = bool
-  description = "Set to true to skip the creation of an IAM authorization policy that permits all event notification instances in the provided resource group reader access to the instance specified in the existing_kms_instance_guid variable."
+  description = "Set this variable value to `true` to skip the creation of an IAM authorization policy that permits all Event Notifications instances in the resource group reader access to the instance specified in the `existing_kms_instance_guid` variable."
```

Comment:
Per the [guidance ](https://test.cloud.ibm.com/docs/solution-as-code?topic=solution-as-code-var-descriptions), use "specify" to describe setting the values. I don't think we need to use "variable value" here.

Reply:
My go-to starter for boolean variables is `Whether...`
But it doesn't work well for this var.
I do usually use `set` for these, so I guess I'm not consistent.

Set to `true` to skip the creation of an IAM authorization policy that permits all Event Notifications instances in the resource group to ... 

What's missing is the default: For example, from other repos: (don't know if it's accurate here.)
> If set to false, specify a value for the KMS instance in the `existing_kms_instance_guid variable`. In addition, no policy is created if `kms_encryption_enabled` is set to `false`."